apps/server/Dockerfile

```diff
@@ -1,22 +1,34 @@
-FROM oven/bun
+FROM oven/bun AS deps
+WORKDIR /app
+
+# Copy only package.json and bun.lockb (if it exists)
+COPY package.json bun.lockb* ./
+
+RUN bun install --frozen-lockfile --ignore-scripts
 
+FROM oven/bun AS builder
 WORKDIR /app
 
+# Copy dependencies from previous stage
+COPY --from=deps /app/node_modules ./node_modules
+
 # Copy project
 COPY . .
 
 # To keep the image small ;)
-RUN rm -rf /app/apps/docs
-RUN rm -rf /app/apps/screenshot-service
-RUN rm -rf /app/apps/web
-RUN rm -rf /app/packages/api
-RUN rm -rf /app/packages/integrations/vercel
-#  Temp solution for bun
-RUN bun install --ignore-scripts
-WORKDIR /app/apps/server
+RUN rm -rf apps/docs apps/screenshot-service apps/web packages/api packages/integrations/vercel
```

Comment:
If we remove them before installing the deps we have a lighter build 
?

Reply:
They aren't needed in the build process correct? I may be wrong but I made that assumption when I saw the folder vercel in there

Reply:
Also the deps come from package.json, that's why it was installed at the top🤔

I do see what you mean, I would need to separate them, keep those not needed in build at the top and delete the rest later